card/spec/lib/card/content_spec.rb

```diff
@@ -306,6 +306,18 @@
         assert_equal 'yo', Card::Content.clean!('<!-- not me -->yo')
         assert_equal 'joe', Card::Content.clean!('<!-- not me -->joe<!-- not me -->')
       end
+
+      it 'fixes regular nbsp order by default' do
+        assert_equal 'space&nbsp; test&nbsp; two&nbsp;&nbsp; space',
+                      Card::Content.clean!('space&nbsp; test &nbsp;two &nbsp;&nbsp;space')
+      end
+
+      it 'doesn\'t fix regular nbsp order with setting' do
+        # manually configure this setting, then make this one live (test above will then fail)
+        pending "Can't set Card.config.space_last_in_multispace= false for one test"
+        assert_equal 'space&nbsp; test &nbsp;two &nbsp;&nbsp;space',
+                      Card::Content.clean!('space&nbsp; test &nbsp;two &nbsp;&nbsp;space')
```

Comment:
I'm not remembering exactly what was happening, but it was a problem changing it.  I think it might be more how it was implemented than how the config setting works.  Once it is used, the code can't change back to the other way of processing.  I think a method is overloaded or something like that.